src/lib/autoinstall/clients/autoyast.rb

```diff
@@ -102,24 +102,45 @@ def main
             }
           },
           "options"    => {
-            "filename"    => { "type" => "string", "help" => "Which profile to use. In " \
-              "check-profile case it supports also remote location like " \
-              "filename=ftp://test.com/example.xml" },
-            "modname"     => { "type" => "string", "help" => "modname=AYAST_MODULE" },
-            "output"      => { "type" => "string", "help" => "where evaluated profile will be " \
-              "written. Default is '~/check_profile_result.xml'. Example 'filename=~/test.xml'." },
-            "run-scripts" => { "type"     => "enum",
-                               "help"     => "run also scripts that are defined in profile. " \
-              "By default false. Example: run-scripts=true",
-                               "typespec" => ["true", "false"] },
-            "import-all"  => { "type"     => "enum",
-                               "help"     => "Do testing import of all sections in profile. " \
-              "Note that scripts are imported when run-scripts is set to true. By default true. " \
-              "Example: import-all=false",
-                               "typespec" => ["true", "false"] }
+            "filename"    => {
+              "type" => "string",
+              "help" => "Which profile to use. In check-profile case it supports also remote " \
+                        "location like filename=ftp://test.com/example.xml"
+            },
+            "modname"     => {
+              "type" => "string",
+              "help" => "modname=AYAST_MODULE"
+            },
+            "output"      => {
+              "type" => "string",
+              "help" => "Where evaluated profile will be written. Default is " \
+                        "'~/check_profile_result.xml'. Example 'filename=~/test.xml'."
+            },
+            "run-scripts" => {
+              "type"     => "enum",
+              "help"     => "Run also scripts that are defined in profile. Be careful when " \
+                            "running pre-installation scripts as root. Use only\n\t\t\t\t     " \
+                            "profiles that you trust. By default false. Example: run-scripts=true",
+              "typespec" => ["true", "false"]
+            },
+            "run-erb"     => {
+              "type"     => "enum",
+              "help"     => "Render the ERB profile. Be careful when running ERB profiles as " \
+                            "root. Use only profiles that you trust. This option is" \
+                            "\n\t\t\t\t     mandatory when running checks for an ERB profile as " \
+                            "root. By defautl true. Example: run-erb=true",
```

Comment:
typo: *default*. And I would say that the default should be 'false', right?

Reply:
The default value is tricky, it depends on the permissions:

* `yast autoyast check-profile filename=profile.erb`. Here `run-erb=true` is assumed because there no risk on running the erb.
* * `sudo yast autoyast check-profile filename=profile.erb`. Here it complains about missing `run-erb` option because there is a risk on running the erb as root.

Does it make sense?

Reply:
I guess I would make the default always the same (to make it simpler). But if we decide to go with different defaults, we should reflect that in the docs too.